docs/topics/mta-web-downloading-an-analysis-report.adoc

```diff
@@ -6,7 +6,7 @@
 [id="mta-web-downloading-an-analysis-report_{context}"]
 = Downloading an analysis report
 
-For your convenience, you can download analysis reports. By default, this option is disabled, but you can it enable by using the {ProductShortName} user interface.
+For your convenience, you can download analysis reports. By default, this option is disabled, but you can enable it by using the {ProductShortName} user interface.
```

Comment:
This whole section is about the UI, so I'd change the "you can enable it by using the {ProductShortName} user interface." part with something like "You can enable it in Administration -> General configuration -> Allow reports to be downloaded after running an analysis."
![image](https://github.com/windup/windup-documentation/assets/117646518/818f4cd9-c30b-4bcc-b358-340a0bdf5ce9)

Reply:
@abrugaro, the thing is just we don't usually describe procedures this way. We do it by using a special procedural block which you can see below this sentence. This sentence is aimed to be an introductory one. I can suggest just  removing the "enable" part since enabling is kind of explained in the procedure itself. We could just rephase this sentence in a manner of the note. For example "Note that by default this option is disabled."

Reply:
Oh ok, that makes sense, thanks!

Reply:
YW :-)